tests/ws-header/ws-header.spec.js

```diff
@@ -0,0 +1,179 @@
+/* eslint-disable no-underscore-dangle, import/extensions, no-proto */
+import {React, render} from 'imports';
+import {WSHeader} from '../../src/index';
+
+/**
+ * Tests for WS-Header
+ * Info: If you want to test the render output after state / props change
+ * you have to do a setTimeout (see other tests) as Jasmine expect is faster then the .render() method
+ */
+describe('Test: <WS-Header />', () => {
+  let container;
+
+  const defaultProps = {
+    setLang: () => {},
+    setLogin: () => {},
+    clientId: null,
+    redirectUrl: '',
+    logoUrl: null,
+    title: '',
+    links: []
+  };
+
+
+  const dummyLinks = [
+    {label: 'Home', value: 'HomeValue', onclick: value => value},
+    {label: 'Link', value: 'LinkValue', onclick: value => value},
+    {label: 'Details', value: 'DetailsValue', onclick: value => value}
+  ];
+
+  const mockAuthResponse = {
+    uid: 'chuckNorrisIsEverywhere',
+    access_token: 'chuckNorrisCanBeAnAccessToken'
+  };
+
+  const mockUserLookUpResponse = {
+    name: 'Chuck Norris',
+    email: 'chuck.norris@zalando.de'
+  };
+
+  const defaultState = {
+    availableLanguages: ['de', 'en']
+  };
+
+  beforeEach(() => {
+    container = document.createElement('div');
+    document.querySelector('body').appendChild(container);
```

Comment:
Addition: If you never attach the elements to the DOM, you don't know if it actually doesn't fail when you do in production.